packages/Input/src/ToggleInputType/ToggleInputType.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment, useState } from 'react';
+
+interface IToggleInputTypeProps {
+  type: string | undefined;
+  toggleType: (newType: string) => void
+}
+
+const ToggleInputType = ({
+  type,
+  toggleType
+}: IToggleInputTypeProps): JSX.Element => {
+
+  interface IPasswordState {
+    icon: string,
+    status: string,
+    text: string
+  }
+
+  const passwordDefaultClasses: IPasswordState = {
+    icon: 'visibility_off',
+    status: 'active',
+    text: 'Show'
+  };
+
+  const [passwordClasses, setPasswordState] = useState(passwordDefaultClasses);
+
+  const handleToggleClasses = (): void => {
+    if (type === 'password') {
```

Comment:
The strings should be a constant, always. We can't leave any hardcoded string anywhere.